src/main/java/in/projecteka/consentmanager/hipLinkInitialization/HIPLinkInitializer.java

```diff
@@ -0,0 +1,8 @@
+package in.projecteka.consentmanager.hipLinkInitialization;
+
+import in.projecteka.consentmanager.hipLinkInitialization.model.request.AuthInitRequest;
+import reactor.core.publisher.Mono;
+
+public interface HIPLinkInitializer {
```

Comment:
Can we think of better name? Its specific to an action .. so maybe HIPLinkInitAction or so?

Reply:
done